public/scripts/ssooverride.js

```diff
@@ -4,14 +4,28 @@
  * the persona SSO that we employ for webmaker.org
  */
 (function() {
-  var hostname = document.getElementById("ssooverride").getAttribute("data-login-hostname");
-  var loginToSave = document.getElementById("ssooverride").getAttribute("data-login-to-save");
+  var overrideElement = document.getElementById("ssooverride");
+  var authHostname = overrideElement.getAttribute("data-login-authHostname");
```

Comment:
they already are. `overrideElement.attributes` is a named item map. it just has a ridiculous API